include/nlohmann/ordered_map.hpp

```diff
@@ -225,6 +230,14 @@ template <class Key, class T, class IgnoredLess = std::less<Key>,
             insert(*it);
         }
     }
+
+private:
+    JSON_HEDLEY_DIAGNOSTIC_PUSH
+#ifdef _MSC_VER
+#pragma warning(disable:4848)
```

Comment:
Only needed if we enable `[[msvc::no_unique_address]]`. (Currently commented out.)